dcos_test_utils/jobs.py

```diff
@@ -0,0 +1,116 @@
+""" Utilities for integration testing metronome in a deployed DC/OS cluster
+"""
+import logging
+
+import retrying
+from requests import HTTPError
+
+from dcos_test_utils.helpers import (ApiClientSession,
+                                     RetryCommonHttpErrorsMixin)
+
+REQUIRED_HEADERS = {'Accept': 'application/json, text/plain, */*'}
+log = logging.getLogger(__name__)
+
+
+class Jobs(RetryCommonHttpErrorsMixin, ApiClientSession):
+    def __init__(self, default_url, session=None):
+        super().__init__(default_url)
+        if session is not None:
+            self.session = session
+        self.session.headers.update(REQUIRED_HEADERS)
+        self._api_version = 'v1'
+
+    @staticmethod
+    def _run_from_history(run_id: str, history: dict) -> dict or None:
```

Comment:
Having this return `None` isn't making a lot of sense to me. Shouldn't we raise an error if we can't find the history that we expect?

Reply:
There is no guarantee that the item is in the list. This is being used - currently - to check history lists, which the `run_id` may not exist in and that is an OK scenario.

Reply:
Given that this is a private method with this one singular use can we just do without this method completely? Its a lot of boilerplate for little value and using in context of the one place is actually called will clarify why it is or isn't raising an exception or returning None when a dict is expected